lib/reesmat.gi

```diff
@@ -246,8 +246,8 @@ function(S, mat)
   
   for x in mat do 
     if ForAny(x, s-> not s in S) then
-      Error("usage: the entries of <mat> must belong to <S>,");
-      return;
+      ErrorNoReturn("the entries of the second argument (a rectangular ", 
+                    "table) must belong to the first argument (a semigroup)");
```

Comment:
I wonder if we could also improve `Error` to print the arg names (e.g. in this case, it might say something like `... in a method for ReesMatrixSemigroup( S, mat )`. Then saying `<mat>` or `<S>` in the error is fine (and *lots* of our error messages refer to args like that).